extensions/amp-analytics/0.1/events.js

```diff
@@ -453,6 +452,13 @@ export class AmpStoryEventTracker extends CustomEventTracker {
   fireListener_(event, rootTarget, config, listener) {
     const type = event['type'];
     const vars = event['vars'];
+    const storySpec = config['storySpec'] || {};
+    const repeat = storySpec['repeat'];
+    const detailsForPage = vars['detailsForPage'];
```

Comment:
Maybe `pageDetails` to be more consistent with the rest of the codebase, and the `storySpec` right above it?